lib/ansible/utils/module_docs_fragments/netapp.py

```diff
@@ -121,24 +121,31 @@ class ModuleDocFragment(object):
   api_username:
     required: true
     description:
-    - The username to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+    - The username to authenticate with the SANtricity Web Services Proxy or Embedded Web Services API.
```

Comment:
I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html